core/src/test/scala/unit/kafka/server/KafkaApisTest.scala

```diff
@@ -105,6 +104,7 @@ class KafkaApisTest {
   private val replicaQuotaManager: ReplicationQuotaManager = EasyMock.createNiceMock(classOf[ReplicationQuotaManager])
   private val quotas = QuotaManagers(clientQuotaManager, clientQuotaManager, clientRequestQuotaManager,
     clientControllerQuotaManager, replicaQuotaManager, replicaQuotaManager, replicaQuotaManager, None)
+  private val quotaCache = new QuotaCache()
```

Comment:
Interestingly, it appears we don't have a test for `handleDescribeClientQuotasRequest()` at all.  Just noting the fact -- nothing to really do unless you want to tackle adding one as part of this PR.